Update key-information.md (#137)
hacks-guide/Guide_Wii: docs/key-information.md

## Console Compatibility

> This guide works with all off-the-shelf retail Wiis, including the Wii U (vWii) and Wii mini.

Reviewer: I would prefer this reads as "the Wii U's virtual Wii environment (vWii)"

In your edit, this reads as "this guide mods Wii U", which it does not.

Author: It is not a virtual Wii environment, saying that would be technically false.

 vWii is just a community-given name, it is not really emulation like VC titles are.
 DS mode on 3DS is also not really virtual.

The focus was on hardware here. Retail Wiis are not software obvs. Wii mini is also hardware and software revision, not the same as Wii in many ways.

I was always more for rephrasing it as Wii mode in the early steps of the guide, since people who didn't touch homebrew before also know it as a Wii mode. **(Like Nintendo did)**

Maybe we can rephrase it as Wii mode on Wii U?
Any other suggestion?

Reviewer: I'm aware of what the vWii is lol. Though "virtual Wii" is *technically* not accurate, it is what vWii stands for.

How about like this?
> This guide works with all off-the-shelf retail Wiis, including the Wii mini and the Wii mode of Wii Us (vWii).

This would be more accurate than how you have currently worded it, and my comment is more meant to call out that your current wording implies this guide mods Wii Us -- it doesn't (as you know).

Author: I just changed the order a bit and left it only as "Wii U's Wii mode". Since it is explained later what **is** vWii, and from that moment we start using vWii in the guide as default. It is good enough.
Tnx for the suggestions, they helped!

Reviewer 2: If we are trying to keep a consistency somewhere then we would have to change everywhere where it says “vWii”

:::: details What if I own a Korean Wii?

> A Korean Wii is fully supported, and can also be region-changed.

Author: You decide, not much of a difference imo https://www.merriam-webster.com/grammar/hyphen-rules-open-closed-compound-words

Exercise caution when modifying region data and follow guides carefully to avoid a brick.

::::

:::: details What if I own a Wii U (vWii)?

Wii mode on Wii U (called vWii in the homebrew community) is fully supported but it is recommended to setup an Aroma environment.
Missing Wii features can be restored. Almost all existing homebrew is currently supported (except for GameCube homebrew).

> Storage should be externally powered since Wii U USB ports are underpowered.

Reviewer: This is only true for Wii U, and as most people using this guide are on a base-model Wii I would advocate against this edit.

Author: Read again. That section **is** meant for Wii U specifically.

Reviewer: Ah, missed that, you are right. Though, I would still re-word this:

> Storage devices will need additional power (either using a Y-cable or an external power source), as the Wii U's USB ports are underpowered.

Author: Great, will update that one!

:::: details What if I own a Wii U (vWii)?

Wii mode on Wii U (called vWii in the homebrew community) is fully supported but it is recommended to setup an Aroma environment.
Missing Wii features can be restored. Almost all existing homebrew is currently supported (except for GameCube homebrew).

> Storage devices will need additional power (either using a Y-cable or an external power source), as the Wii U's USB ports are underpowered.

Reviewer 2: ```suggestion
Storage devices will need additional power (either using a Y-cable or an external power source), as the Wii U's USB ports do not provide sufficient power on their own.
```

Its network hardware is based on 802.11g (retroactively named WiFi 3), which operates on the 2.4 GHz band. It also supports a maximum of WPA2-PSK (AES) personal network encryption. You must make sure your network supports these standards if you would like to use WiFi with your Wii.

For the Wii U, vWii uses the internet settings set in the Wii U's main settings, but can be optionally separated with the [Wii VC Launch](https://hb-app.store/wiiu/WiiVCLaunch) Aroma plugin.

Please consult [the network FAQ](faq#networking-faq) for more information.

## Getting Help

> Ran into an issue while following the guide, already have an issue on your modded Wii, or just a plain need for answers?

Reviewer: This is grammatically incorrect, the original wording is not.

Parallelism: "ran", "have", and "need" are the verbs here. They all agree with each other. In your edit, parallelism is broken.
